src/getStorage.js

```diff
@@ -0,0 +1,4 @@
+export default function getStorage() {
+  const storage = JSON.parse(localStorage.getItem('storage'));
+  return (storage !== null || storage !== undefined) ? storage : [];
+}
```

Comment:
- [ ] As per project requirements we have to store the updated array of items in local storage, so the user gets the correct list values after the page reloads. But after marking the task as completed and reload the page than it comes to the original position. Kindly try to fix this issue when the user refresh the page it should render updated array item